### PR TITLE
feat(company): hero section

### LINKED ISSUE #22: feat: company page hero section
image

### PR DESCRIPTION
**Pull request by Im-Kunal-13**
closes #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,31 +22,32 @@
...
 		"@types/react-google-recaptcha": "^2.1.5",
-		"eslint": "8.21.0",
-		"eslint-config-next": "12.2.4",
+		"eslint": "^8.21.0",
+		"eslint-config-next": "^12.2.4",
 		"husky": "^8.0.1",
-		"lint-staged": ">=10",
-		"prettier": "2.7.1",
-		"typescript": "4.7.4"
+		"lint-staged": "^13.0.3",
```
Comment (ashuvssut) [inline package.json:48]: lint-staged v13 has issues with the git-hooks that we are using. Please revert all dependency version changes. Keep "swiper" package. 

```
# pull my changes done in your PR
git pull https://github.com/Im-Kunal-13/walking-pal-web.git  company-hero

# go to first commit
git reset --soft HEAD~2

#unstage those 2 files
git reset HEAD -- package-lock.json
git reset HEAD -- package.json

yarn add swiper

git add .

git commit -m "feat(company): hero section"

git push -f https://github.com/Im-Kunal-13/walking-pal-web.git  company-hero 

```

File: pages/company/Company.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+				style={{
+					display: "flex",
+					flexDirection: "column",
+					justifyContent: "center",
+					alignItems: "center",
+				}}
+			>
+				<Hero />
+				{/* <Gallery />
+				<OurStory /> */}
```
Comment (ashuvssut) [inline pages/company/Company.tsx:32]: Uncomment these two

**Review by ashuvssut — CHANGES_REQUESTED** [review]

File: components/Header/Header.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ import { HeaderMobile } from "components/Header/components/HeaderMobile";
 export const links = [
 	{ name: "home", route: "/" },
 	{ name: "about", route: "/about" },
+	{ name: "company", route: "/company" },
```
Comment (ashuvssut) [inline components/Header/Header.tsx:11]: remove this. company page is not ready to be shown to users. we will add this when company page finishes

File: pages/company/company.module.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.hero_img_container {
+	width: "100%";
+	display: flex;
+	justify-content: center;
+}
```
Comment (ashuvssut) [inline pages/company/company.module.scss:5]: remove all these styles. add these as `system` props to `<Box>` component

File: pages/company/components/Hero.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+						disableOnInteraction: false,
+						pauseOnMouseEnter: true,
+					}}
+					modules={[Autoplay]}
+					centeredSlides={true}
+				>
+					{[company4, company1, company2, company3, company5].map(
+						(img, index) => (
+							<SwiperSlide key={index}>
+								<div className={styles.hero_img_container}>
```
Comment (ashuvssut) [inline pages/company/components/Hero.tsx:50]: replace `div` with this

```jsx
<Box width="100%" display="flex"	justifyContent="center">
```

**Review by ashuvssut — CHANGES_REQUESTED** [review]

**Comment by ashuvssut** [comment]
Before making all these changes, update ur PR branch. (pull from upstream dev) then Resolve these merge conflicts

File: package.json (hunk 1 above)
Comment (Im-Kunal-13) [reply package.json:48]: > 

Ok. On it

File: pages/company/Company.tsx (hunk 2 above)
Comment (Im-Kunal-13) [reply pages/company/Company.tsx:32]: > Uncomment these two

These are not ready and are copied from the about page just for reference

Comment (ashuvssut) [reply pages/company/Company.tsx]: Oh my bad. 

in that case, you will recieve such 2 lines when you pull from dev. 2 components (`<Data>`, `<Story>`)are already added by other 2 contributors.

**Pull request merged**